Pipfile

```diff
@@ -25,6 +23,8 @@ mock = "*"
 types-mock = "*"
 types-PyYAML = "*"
 pytest-asyncio = "*"
+pytest-pydocstyle = "*"
+charset-normalizer = "*"
```

Comment:
Do we need to add this?

Reply:
It wouldn't install `black` for me without `charset-normalizer` installed, i guess when i installed it manually it added it to the Pipfile

Reply:
Gonna' revert this here, we can add it as another PR if the issue is replicable